lib/routes/About/AboutRoute.tsx

```diff
@@ -20,6 +21,7 @@ const html: Record<IPossibleLanguages, string> = {
   "es": converter.makeHtml(AboutEsMarkdown),
   "fr": converter.makeHtml(AboutFrMarkdown),
   "ru": converter.makeHtml(AboutRuMarkdown),
+  "de": converter.makeHtml(AboutDeMarkdown),
```

Comment:
Sorry about that. And also the quotation marks. :flushed:

Reply:
No worries! I usually ask people to just give me the files and I import them myself 😅  

Just the fact that you did 98% of the job yourself takes a lot of work off my shoulder.